indico/modules/events/management/templates/contribution_person_list.html

```diff
@@ -68,19 +67,17 @@
                                 -
                             {% endif %}
                         </td>
-                        {% if include_submitters %}
-                            {% set is_submitter = person_roles.submitter %}
-                            <td class="i-table submitter-column">
-                                {%- if is_submitter %}
-                                    {% set submitter_title = _("This person submitted an abstract") %}
-                                {%- else -%}
-                                    {% set submitter_title = _("This person hasn't submitted any abstracts") %}
-                                {% endif -%}
-                                <i class="icon-checkmark submitter-icon {{ 'inactive' if not is_submitter }}"
-                                   title="{{ submitter_title }}"></i>
-                            </td>
-                        {% endif %}
-                        <td class="i-table speaker-column">
+                        <td class="i-table roles-column">
+                            {% if include_submitters %}
+                                {% set is_submitter = person_roles.submitter %}
+                                    {%- if is_submitter %}
```

Comment:
too much indentation; `{% set foo = ... %}` doesn't start a block so there's no extra level of indentation after that